vperfetto-min.cpp

```diff
@@ -0,0 +1,148 @@
+#include "perfetto.h"
+#include "vperfetto-min.h"
+#include "vperfetto.h"
+
+#include <string>
+#include <thread>
+#include <fstream>
+#include <unordered_map>
+
+PERFETTO_DEFINE_CATEGORIES(
+    ::perfetto::Category("gfx")
+        .SetDescription("Events from the graphics subsystem"));
+PERFETTO_TRACK_EVENT_STATIC_STORAGE();
+
+#define TRACE_COUNTER(category, name, value)  \
+    PERFETTO_INTERNAL_TRACK_EVENT(                  \
+            category, name, \
+            ::perfetto::protos::pbzero::TrackEvent::TYPE_COUNTER, [&](::perfetto::EventContext ctx){ \
+            ctx.event()->set_counter_value(value); \
+            })
+
+#ifdef __cplusplus
```

Comment:
Guess this will be always c++?

Reply:
Ah yup. Removed the ifdef in 6e62f3c234687cf3800f7455f16ef066955fe22a